cou/cli.py

```diff
@@ -33,6 +33,9 @@
 COU_DIR_LOG = pathlib.Path(os.getenv("HOME", ""), ".local/share/cou/log")
 AVAILABLE_OPTIONS = "cas"
 
+# NOTE(gabrielcocenza) top-level module to reference the same logger object across modules.
```

Comment:
is this accurate? the various submodules use `logger = logging.getLogger(__name__)` which should yield submodule-specific names (ftr, I think either a single logger name or submodule-specific logger names are fine)

Reply:
See the comment bellow why I used `cou` here. Otherwise the other modules won't take this logger object as reference

Reply:
I've removed this part and now the logger is using `logging.getLogger(__name__)`